module/BeadPoolManifest.py

```diff
@@ -179,6 +184,71 @@ def from_string(source_strand):
             raise ValueError(
                 "Unexpected value for source strand " + source_strand)
 
+
+class IlmnStrand(object):
+    Unknown = 0
+    TOP = 1
+    BOT = 2
+    PLUS = 1
+    MINUS = 2
+
+    @staticmethod
+    def to_string(ilmn_strand):
+        """Get an integer representation of Illumina strand annotation
+        
+        Args:
+            ilmn_strand (str) : string representation of Illumina strand annotation (e.g., "T")
+            
+        Returns:
+            int : int representation of Illumina strand annotation (e.g. IlmnStrand.TOP)
+        
+        Raises:
+            ValueError: Unexpected value for Illumina strand
+        """
+        if ilmn_strand == IlmnStrand.Unknown:
+            return "U"
+        elif ilmn_strand == IlmnStrand.TOP:
+            return "T"
+        elif ilmn_strand == IlmnStrand.BOT:
+            return "B"
+        elif ilmn_strand == IlmnStrand.PLUS:
+            return "P"
+        elif ilmn_strand == IlmnStrand.MINUS:
+            return "M"
+        
+        else:
+            raise ValueError(
+                "Unexpected value for ilmn strand " + ilmn_strand)
+
+    @staticmethod
+    def from_string(ilmn_strand):
+        """
+        Get a string representation of Illumina strand annotation
+        
+        Args:
+            ilmn_strand (int) : int representation of Illumina strand (e.g., IlmnStrand.TOP)
+        
+        Returns:
+            str : string representation of Illumina strand annotation
+        
+        Raises:
+            ValueError: Unexpected value for Illumina strand
+        """
+        if ilmn_strand == "U" or ilmn_strand == "":
```

Comment:
Shouldn't empty string also raise a ValueError?

Reply:
Right, empty string is now an exception.

Question though:
for class RefStrand it states:
        if ref_strand == "U" or ref_strand == "":
            return RefStrand.Unknown

Is that the desired behavior for empty string?